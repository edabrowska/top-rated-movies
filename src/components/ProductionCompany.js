import React from 'react'
import bemCx from 'bem-modifiers'

const ProductionCompany = ({ logo, name }) => {
  return (
    <div className={bemCx('production-company', !logo && 'without-logo')}>
      {logo && <div className='production-company__logo-wrapper'>
        <img
          src={`https://image.tmdb.org/t/p/w500/${logo}`}
          alt={`${name} logo`}
          className='production-company__logo'
        />
      </div>}
      <p className='production-company__name'>{name}</p>
    </div>
  )
}

export default ProductionCompany
