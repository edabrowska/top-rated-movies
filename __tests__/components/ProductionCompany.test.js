import React from 'react'
import renderer from 'react-test-renderer'

import ProductionCompany from '../../src/components/ProductionCompany'

describe('ProductionCompany', () => {
  it('Snapshot', () => {
    const component = renderer.create(
      <ProductionCompany
        logo={null}
        name="disney"
      />
    )
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
