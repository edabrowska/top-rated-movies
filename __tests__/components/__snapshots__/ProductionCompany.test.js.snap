// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductionCompany Snapshot 1`] = `
<div
  className="production-company production-company--without-logo"
>
  <p
    className="production-company__name"
  >
    disney
  </p>
</div>
`;
